.github/workflows/main.yml

```diff
@@ -29,7 +30,9 @@ jobs:
     - name: Maven Version
       run: mvn --version
     - name: Build with Maven
-      run: mvn clean -B package --file pom.xml
+      run: mvn clean package -B --file pom.xml -Dnpm.skip -DskipTests
```

Comment:
Why do you skip NPM build? If there's a problem with that the check won't figure out.

Reply:
To speed up the build. I can remove it though.